en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling.md

```diff
@@ -1,12 +1,12 @@
-# Distributed throttling
+# Distributed Rate Limiting
 
-To learn about the concepts of throttling in Choreo Connect, please follow the [APIM As Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/#rate-limiting).
+To learn about the concepts of Rate Limiting (Throttling) in Choreo Connect, see [API-M As a Control Plane - Rate Limiting]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/#rate-limiting).
 
-## Enable Distributed Throttling
+## Enable Distributed Rate Limiting
 
-Throttling (Rate Limiting) in Choreo Connect works with API Manager (Traffic Manager).
+Rate Limiting in Choreo Connect works with API Manager (Traffic Manager).
 
-To enable distributed throttling, change the below configurations in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file as required.
+To enable distributed Rate Limiting, change the following configurations in the `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf/config.toml` file as required.
 
 ``` toml
```

Comment:
@pubudu538 , please check this config. I had a merge conflict on this.
![Mergely-Diff-online-merge-documents](https://user-images.githubusercontent.com/5195851/142986698-dbe7989a-dc58-47c9-a347-5754df054f0a.png)